assets/js/modules/analytics/datastore/settings.js

```diff
@@ -198,9 +219,10 @@ export function validateCanSubmitChanges( select ) {
 		INVARIANT_INVALID_ACCOUNT_ID
 	);
 
+	const isUAEnabled = select( CORE_FORMS ).getValue( FORM_SETUP, 'enableUA' );
 	// Do not require selecting a property or profile if `ga4Reporting` is enabled.
-	// Therefore, only validate these if `ga4Reporting` is not enabled.
-	if ( ! isFeatureEnabled( 'ga4Reporting' ) ) {
+	// Therefore, only validate these if `ga4Reporting` is not enabled OR `enableUA` is enabled.
+	if ( ! isFeatureEnabled( 'ga4Reporting' ) || isUAEnabled ) {
```

Comment:
```suggestion
	if ( ! ga4ReportingEnabled || isUAEnabled ) {
```

Reply:
Due to single usage, I didn't declare a variable named `ga4ReportingEnabled` in the `validateCanSubmitChanges` function.

Reply:
Ohhh, I thought it was declared earlier due to the GitHub diff view. My bad! 👍🏻